src/petals/server/server.py

```diff
@@ -167,6 +170,22 @@ def __init__(
 
         self.stop = threading.Event()
 
+    def _choose_num_blocks(self) -> int:
+        assert self.converted_model_name_or_path != "bigscience/bloom-petals", \
```

Comment:
We can make a generic solution for all models later (e.g., by creating a fake block inside `with accelerate.no_init_weights():` and counting its parameters)